### ISSUE TITLE
Phase 1 feature tracker

### ISSUE THREAD

**Issue by psiinon** [zaproxy/zap-hud#89]
The features we really want before we can start asking people to beta test it:

- [x] Show alerts (using events)
- [x] Spider (using events)
- [x] Active scan (using events)
- [x] Show / hide (client side)
- [x] Highlight alerts on target domain (client side)
- [x] Break (using events)
- [x] Show history
- [ ] Manage sessions
- [ ] Manage contexts
- [ ] Ajax Spider

Nice to haves

- [ ] Advanced options for active scan, spider etc
- [ ] Search messages
- [ ] Show websocket messages
- [ ] Break on websocket messages
- [ ] HTTP session management
- [x] Show sites tree
- [ ] Fuzzer

Feel free to add to this :)
